tm2/pkg/sdk/auth/params_test.go

```diff
@@ -123,9 +123,7 @@ func TestParamsString(t *testing.T) {
 	for _, tt := range cases {
 		t.Run(tt.name, func(t *testing.T) {
 			got := tt.params.String()
-			if diff := cmp.Diff(got, tt.want); diff != "" {
-				t.Fatalf("Mismatch: got - want +\n%s", diff)
-			}
```

Comment:
cmp.Diff is very useful in debugging hairy cases where you need to squint as much which is why it is used in those cases for example even in rpc/lib/server: it is meant to make maintenance for other developers very trivial.

Reply:
Yeah, but `assert.Equal` already shows a diff on an error condition, so using cmp seems redundant.